packages/fxa-content-server/scripts/test-ci.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+
+DIR=$(dirname "$0")
+
+function test_suite() {
+  local suite=$1
+  local numGroups=$2
+
+  for i in $(seq "$numGroups")
+  do
+    node tests/intern.js --suites="${suite}" --output="../../artifacts/tests/${suite}-${numGroups}-${i}-results.xml" --groupsCount="${numGroups}" --groupNum="${i}" || \
+    node tests/intern.js --suites="${suite}" --output="../../artifacts/tests/${suite}-${numGroups}-${i}-results.xml" --groupsCount="${numGroups}" --groupNum="${i}" --grep="$(<rerun.txt)"
+  done
+}
+
+cd "$DIR/.."
+
+test_suite circle 3
```

Comment:
We are not using this script anywhere correct?  I don't understand why we are restoring this.